modules/RTC/JitsiLocalTrack.js

```diff
@@ -134,6 +131,48 @@ JitsiLocalTrack.prototype.isEnded = function () {
 };
 
 /**
+ * Sets handlers to the MediaStreamTrack object that will detect camera issues.
+ */
+JitsiLocalTrack.prototype._initNoDataFromSourceHandlers = function () {
+    if(this.isVideoTrack() && this.videoType === VideoType.CAMERA) {
+        this._setHandler("track_mute", () => {
+            if(this._checkForCameraIssues()) {
+                let now = window.performance.now();
+                this._noDataFromSourceTimout = setTimeout(() => {
+                    this._noDataFromSourceTimout = null;
+                    //unset the handler
+                    this._setHandler("track_unmute", undefined);
+                    if(this._checkForCameraIssues())
+                        this._fireNoDataFromSourceEvent();
+                }, 3000);
+                this._setHandler("track_unmute", () => {
+                    if(this._noDataFromSourceTimout) {
```

Comment:
Since the `track_unmute` handler is unset after `this._noDataFromSourceTimout = null`, the check `this._noDataFromSourceTimout` is superfluous and forces another level of indentation. I think the source code will be simplified by removing that check.